addons/block_code/blocks/lifecycle/process.tres

```diff
@@ -5,11 +5,12 @@
 [resource]
 script = ExtResource("1_pmina")
 name = &"process"
+target_node_class = ""
 description = "Attached blocks will be executed during the processing step of the main loop"
 category = "Lifecycle"
 type = 1
 variant_type = 0
-display_template = "On Process"
+display_template = "Every frame"
```

Comment:
Are we going to consistently use *Sentence case* rather than *Title case* for blocks?

I checked what MakeCode Arcade does and actually it uses *lower case*!

Reply:
Yes, I proposed lower case once to the other Will. Today we have a mix of Title case for short labels ("Viewport Center") and Sentence case for long labels. I just checked and Scratch has all in lower case too. So unless anyone has a strong argument I'll switch all to lower case.